recipes-bsp/atf/atf_git.bbappend

```diff
@@ -0,0 +1,8 @@
+FILESEXTRAPATHS_append_lx2160acex7 := "${THISDIR}/${PN}-lx2160acex7:"
+
+SRC_URI_append_lx2160acex7 = "\
+	file://0001-plat-nxp-Add-lx2160acex7-module-support.patch \
+	file://0002-plat-nxp-lx2160a-auto-boot.patch \
+"
+
+# ddrphyopt_lx2160acex7 = "fip_ddr_sec"
```

Comment:
This is commented code, why?

Reply:
I added this very early and forgot to cleanup. Maybe this becomes necessary again when using secure-boot.